lib/src/Uncertainty/Algorithm/Transformation/MarginalTransformation/MarginalTransformationEvaluation.cxx

```diff
@@ -187,7 +187,8 @@ void MarginalTransformationEvaluation::initialize(const Bool simplify)
         {
           OSS oss;
           oss.setPrecision(20);
-          oss << "max(0.0, " << gamma2 + log(lambda2) << " + " << lambda1 << " * ";
+          // T(x) = gamma2+lambda1/lambda2*(x-gamma1)
+          oss << "max(0.0, " << gamma2 << " + " << lambda1 / lambda2 << " * ";
           if (gamma1 == 0.0) oss << xName << ")";
```

Comment:
maybe a simplification when gamma2==0.0 (common case) or when lambda1==lambda2?

Reply:
imho this is over-optimizing, and worse, this may lead potential bugs in unused code branches
I'd have coded this just like this:
```oss << "max(0.0, " << gamma2 << " + " << lambda1 / lambda2 << " * (xName - "<< gamma1<<")";```

Reply:
@jschueller The objective is not to optimize the evaluation of the transformation but rather to have something nice to show to the user. When you print a function and get '0.0+1.0*x' it hurts a little bit the eyes...

Reply:
ok